src/objects/core/zcl_abapgit_filename_logic.clas.abap

```diff
@@ -166,12 +168,12 @@ CLASS ZCL_ABAPGIT_FILENAME_LOGIC IMPLEMENTATION.
 
     CLEAR es_item.
     es_item-obj_type = lv_type.
-    es_item-obj_name = lv_name.
+    es_item-obj_name = to_upper( lv_name ).
 
     " Get mapping specific to object type
     map_filename_to_object(
       EXPORTING
-        iv_filename = iv_filename
+        iv_filename = lv_name " original-cased object name part only
```

Comment:
rename the parameter to reflect what it contains?

and I guess SICF still works as it only uses the object name part of the filename

Reply:
Do you mean the interface parameter?

Yes, sicf just using first 25 chars of the passed string, so no changes needed there

Reply:
Also what should be a good name here. In fact 'filename' is the right term, it just wasn't used super accurately because it is very univerdal so now it's confusing. Iv_obj_filename? Iv_obj_name_prefix? Iv_item_filename (my best option)?

Reply:
What about this? If OK i can also change in the interface